life/about-me-instructions.md

```diff
@@ -0,0 +1,24 @@
+# About Me Sections
+
+I'm a big fan of self-care, and I think part of that involves recognizing that you can still learn something about yourself and acknowledging your happies and crappies. This section is specifically for those kinds of TILs.
+
+I have a separate branch on this for my own personal use that at least for the time being, I don't plan on pushing. That said, I wanted to add this because I'm getting a lot out of it and wanted to share the idea.
```

Comment:
eh, I went ahead and made an about-me directory anyway. Once this gets merged then contributors can just stick their about-me files in there and it's all good